src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Abs_ConsistentOneColumnTableResult.txt

```diff
@@ -0,0 +1,10 @@
+﻿#SETUP: ConsistentOneColumnTableResult
+
+>> Abs([-1, 0, 123456.789, -123456.789])
```

Comment:
>[-1, 0, 123456.789, -123456.789] [](http://example.com/codeflow?start=7&length=32)

This test works even without the ConsistentOneColumnTableResult feature on. A better test would be to use a different input schema, such as `Abs(Table({a:-1},{a:0},{a:123456.789},{a:-123456.789}))`